Cargo.toml

```diff
@@ -21,43 +21,37 @@ exclude = [
 ]
 
 [dependencies]
-aead = "0.4"
-async-trait = "0.1"
-byteorder = "1.3"
-bytes = "1.0"
-chacha20poly1305 = "0.8"
-ed25519-dalek = { version = "1.0", features = ["std", "serde"] }
-erased-serde = "0.3"
+aead = "0.4.3"
+async-trait = "0.1.35"
+byteorder = "1.4.3"
+bytes = "1.1.0"
+chacha20poly1305 = "0.9.0"
+ed25519-dalek = { version = "1.0.1", features = ["std", "serde"] }
+erased-serde = "0.3.20"
 eui48 = { version = "1.0", features = ["serde"] }
 futures = "0.3"
 get_if_addrs = "0.5"
 hkdf = "0.11"
-hyper = { version = "0.14", features = ["server", "http1"] }
+hyper = { version = "0.14.18", features = ["server", "http1"] }
 libmdns = "0.6"
-log = "0.4"
-num = "0.2"
+log = "0.4.17"
+num = "0.4.0"
```

Comment:
IIRC there was a reason I didn't update some of the dependencies, e.g. `chacha20poly1305` and `num`, yet, since there was something breaking on the pairing step handlers I didn't bother to look into yet. Did you test a whole pairing process after all these dependency updates? Is everything still working?